Makefile

```diff
@@ -75,7 +75,7 @@ install-dev: virtualenv
 
 # this only works outside the venv
 virtualenv:
-	$(PYTHON) -m venv $(VENV) --prompt $(PENV) --system-site-packages
+	$(PYTHON) -m venv $(VENV) --prompt $(PENV)
```

Comment:
@lucpeterson you're right this would force folks to download more libraries. I removed it because on Quartz it was using the wrong version of Sphinx, which was stopping me from updating the docs.

These Makefile targets are pretty much only used by us, so whatever works best for us is fine by me.

Reply:
For now I'll merge into `develop` with this reverted.